README.md

```diff
@@ -1,3 +1,24 @@
 # Hello-Microverse
 
-First Microverse Project
+> This is my first Microverse project.
+
+## The objectives of this projects are:
+
+- Create branches with different base branches.
+- Understand the concept of version control.
+- Follow Gitflow.
+- Open pull requests in GitHub.
+- Write descriptive, easy to understand commit messages.
+- Write short, detailed, easy to understand descriptions of pull requests.
+- Write a descriptive and easy to understand README in business English for every project submitted.
+- Demonstrate an ability to submit a project for code review.
+
```

Comment:
- [ ] Your readme file looks great and you used Microverse template very good job :+1: . However there is one small thing that you should also add:
_Please include the installation section, where you should explain to other people how can they clone and run your project_. In case you need a refreshment check the [Microverse template](https://github.com/microverseinc/readme-template)

Reply:
Thanks for the feedback
I've fixed the issue that you pointed out